tools/markdown-magic/src/transforms/exampleGettingStartedTransform.cjs

```diff
@@ -44,10 +48,10 @@ const generateExampleGettingStartedSection = (
 		`1. Run \`pnpm start\` from this directory and open <http://localhost:8080> in a web browser to see the app running.`,
 	);
 
-	return formattedSectionText(
-		sectionBody.join("\n"),
-		includeHeading ? "Getting Started" : undefined,
-	);
+	return formattedSectionText(sectionBody.join("\n"), {
+		...headingOptions,
```

Comment:
With this code, shouldn't we expect the entire `README.md` documents to have `h2` styling for `Getting Started` section (since `h2` is the default)?

Below items still have `h1` styling applied and I can't find if `h1` style is given:
[fluid-telemetry/Getting Started](https://github.com/Josmithr/FluidFramework/blob/docs/up-markdown-documenter/packages/framework/client-logger/fluid-telemetry/README.md#getting-started)

[experimental-tree/Getting Started](https://github.com/Josmithr/FluidFramework/blob/docs/up-markdown-documenter/experimental/dds/tree/README.md)

Reply:
Those documents aren't using these templates; those are just hand-written.

Reply:
All of these templates are opt in using syntax like:
```md
<!-- AUTO-GENERATED-CONTENT:START (README_EXAMPLE_GETTING_STARTED_SECTION:headingLevel=2&usesTinylicious=FALSE) -->
```